package.json

```diff
@@ -114,7 +114,7 @@
 		"@hashicorp/react-code-block": "^6.2.2",
 		"@hashicorp/react-command-line-terminal": "^3.0.0",
 		"@hashicorp/react-consent-manager": "^9.0.1",
-		"@hashicorp/react-docs-page": "^17.7.2",
+		"@hashicorp/react-docs-page": "^17.8.0-canary-20230411190718",
```

Comment:
@nayres Let's get publish `17.8.0` (stable) published, update this, and then this PR should be good to go!